elephant/signal_processing.py

```diff
@@ -134,6 +135,111 @@ def zscore(signal, inplace=True):
         return result[0]
     else:
         return result
+    
+    
+def cross_correlation_function(signal, ch_pairs, dt=1., env=False, nlags=None):
+
+    """
+    Computes unbiased estimator of the cross-correlation function.
+    
+    Calculates the unbiased estimator of the cross-correlation function 
+    R(tau) = 1/(N-|k|) R'(tau), where R'(tau) = E[x(t)*y(t+tau)] for signal1 
+    and signal2 in a pairwise manner, i.e. signal1[0] vs signal2[0], signal1[1] 
+    vs signal2[1] and so on. The cross-correlation function is obtained 
+    by np.correlate. signal1 and signal2 are zscored beforehand.
```

Comment:
You seem not to use ```np.correlate```

Reply:
description has been changed accordingly